roachtest: update version map and fixtures for v21.1.0

celiala opened the pull request:
Update version to 21.1.0 for version map
and fixtures.

Release note: None (testing change)

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/65376)
<!-- Reviewable:end -->

celiala committed:
roachtest: update version map and fixtures

Update version to 21.1.0 for version map
and fixtures.

Release note: None (testing change)

celiala commented:
TFTR!

bors r+
